Tests/test_content.py

```diff
@@ -899,6 +899,7 @@ def execute_testing(tests_settings, server_ip, mockable_tests_names, unmockable_
                 updating_mocks_msg = "Pushing new/updated mock files to mock git repo."
                 prints_manager.add_print_job(updating_mocks_msg, print, thread_index)
                 ami.upload_mock_files(build_name, build_number)
+                prints_manager.execute_thread_prints(thread_index)
```

Comment:
why not execute it directly under the finally? why specifically for AMI on the master branch?

Reply:
not sure I understand this comment.

Reply:
why is that in the condition? why not run at any time in the `finally` block